Lib/test/test_dataclasses.py

```diff
@@ -2855,10 +2854,13 @@ class C:
 
 class TestStringAnnotations(unittest.TestCase):
     def test_classvar(self):
+        # Some expressions recognized as ClassVar really aren't.  But
+        #  if you're using string annotations, it's not an exact
+        #  science.
         # These tests assume that both "import typing" and "from
         # typing import *" have been run in this file.
         for typestr in ('ClassVar[int]',
-                        'ClassVar [int]',
+                        'ClassVar [int]'
```

Comment:
This seems like a bug

Reply:
Yes, this is a bug that I accidentally stumbled against in the initial commit and fixed. I don't think we need to revert that